subsys/net/lib/dhcpv4/Kconfig

```diff
@@ -44,6 +44,22 @@ config NET_DHCPV4_ACCEPT_UNICAST
 	  If set, the network stack will accept unicast DHCPv4 responses from
 	  servers, before the assigned address is configured on the interface.
 
+config NET_DHCPV4_VENDOR_CLASS_IDENTIFIER
+	bool "Send vendor class identifier in DHCPv4 request"
+	depends on NET_DHCPV4
+	default n
+	help
+		If set, the network stack will include the specified string in the
+		DHCPv4 vendor class identifier option in the DHCPv4 request.
```

Comment:
Please indent the same way as other help options, there should be two space indentation for the help string.